acl/tests/test.bats

```diff
@@ -0,0 +1,31 @@
+failWithMessage() {
+    local rest="$@"
+    diff <(echo "actual result is below") <(echo "${rest}")
+}
+
+# parse the package identifier
+IFS=/ read -r -a _pkg_ident_array <<< "${TEST_PKG_IDENT}"
+_origin="${_pkg_ident_array[0]}"
+_package="${_pkg_ident_array[1]}"
+_version="${_pkg_ident_array[2]}"
+_timestamp="${_pkg_ident_array[3]}"
+
+@test "${TEST_PKG_IDENT} has a valid installation directory" {
```

Comment:
Testing for a valid installation directory isn't something we should be doing.

Reply:
Removed.